doc/usage.rst

```diff
@@ -519,7 +520,8 @@ project key or id in case the regular expression doesn't come up with a match or
 
 ``item_to_ticket_regex`` defines the regular expression used to filter item IDs to be exported as Jira tickets.
 A warning gets reported when a Jira ticket already exists. These warnings can be disabled by setting
-``warn_if_exists`` to ``True``.
+``warn_if_exists`` to ``True``. Exceptions raised by this feature can be converted to warnings by means setting the
```

Comment:
```suggestion
``warn_if_exists`` to ``True``. Errors raised by this feature can be converted to warnings by means setting the
```

Reply:
Doesnt make sense though, it says warn if exists...

Reply:
By *this feature* I meant the entire *Jira ticket creation* feature. I'll clarify this section.

Reply:
Done in 0f66bdf44701fba465e6de4811c8cb05cf0d5dcf